eiffel-vocabulary/EiffelActivityQueuedEvent.md

```diff
@@ -22,8 +22,13 @@ __Description:__ The circumstances triggering the activity.
 #### data.trigger.type
 __Type:__ String  
 __Required:__ Yes  
-__Legal values:__ MANUAL, EIFFEL_EVENT, REBUILD, SOURCE_CHANGE, TIMER, OTHER  
-__Description:__ The type of trigger.
+__Legal values:__ MANUAL, EIFFEL_EVENT, SOURCE_CHANGE, TIMER, OTHER  
```

Comment:
Arguably something of a corner case, but a fair point. I don't see any reason to not make trigger an array.